app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<h1> Pending invitations </h1>
+  <ul>
+    <% @f.each do |x| %>
+      <li>
+        <% if x.confirmed == false %>
+          <p><%= x.requester.name %></p>
+          <div class='btn-div'>
+          <%= form_for(:friendships, url: friendship_path(@i), method: :patch ) do |f| %>
+            <%#= f.hidden_field :confirmed %>
+            <div>
+              <%= f.submit 'Accept' %> 
+            </div>
+          <% end %>
+          <%= form_for(:friendships, url: friendship_path(@i), method: :delete ) do |f| %>
+            <%= f.hidden_field :confirmed %>
+
+            <div>
+              <%= f.submit 'Reject', class: 'btn-right'%>
+            </div>
+          <% end %>
+
+          </div>
+        <% end %>
```

Comment:
- [ ] Instead of having the if/end statement create a helper method that has the if and renders a partial that contains the markup within the if/end statement instead of having the logic in the view